docs/source/deprecation.md

```diff
@@ -13,7 +13,11 @@ the leapp standard library, etc. But don't cover deprecated functionalities
 from particular leapp repositories (e.g. the [elt7toel8](https://github.com/oamg/leapp-repository/tree/master/repos/system_upgrade/el7toel8) leapp repository). For
 such information, see [Deprecated functionality in the el7toel8 repository](el7toel8/deprecation.html#deprecated-functionality-in-the-el7toel8-repository).
 
-*Note: No new deprecation present yet.*
+## v0.14.0 <span style="font-size:0.5em; font-weight:normal">(till September 2022)</span>
```

Comment:
@fernflower  deprecation is usually 6m since the next release. this feature has not been deprecated with the release 0.14.0 but it will be deprecated with the next release.  so it's expected something like:

```suggestion
## current upstream development <span style="font-size:0.5em; font-weight:normal">(till the next release + 6months)</span>
```

the doc needs to be updated then with the new release PR